src/ucm/util/sys.h

```diff
@@ -88,4 +90,19 @@ void ucm_prevent_dl_unload();
 char *ucm_concat_path(char *buffer, size_t max, const char *dir, const char *file);
 
 
+
+BEGIN_C_DECLS
```

Comment:
can this be at the beginning of the file?

Reply:
Actually this results in a number of compilation errors.

Reply:
why do we even need this? are we including from C++?

Reply:
Thanks. Done.